### ISSUE TITLE
mistake on error.h ?

### ISSUE THREAD

**Issue by supernono-fr** [speeduino/speeduino#1015]
hello 

i think there is a mistake on file  error.h on line 24 

#define ERR_DEFAULT_CKT_SHORT   80 //Note that the default is 40C. 80 is used due to the -40 offset

i think CKT is wrong, CLT must be right

**Comment by noisymime** [comment]
Yes, this is correct however `ERR_DEFAULT_CKT_SHORT` (along with a number of other defines in this file) are not actually used anywhere. Ideally this would get fixed in the redesign of the whole error system, but I haven't made much of a start on this yet. 

I'll fix the typo on this for now, but no impact from the error at all.

**Comment by supernono-fr** [comment]
OK thank you for your answer, realy interested by the error system can i do something to help ?

**Comment by adbancroft** [comment]
Closing - non-issue that's out of dare
